### PR TITLE
Fix ChromeLogger

### PR DESCRIPTION
**Pull request by dafriend**
**Description**
Fixes Issue #2616

### CODE REVIEW

**Comment by MGatner** [comment]
Thanks for taking a pass at this! Looks like a test just needs to be updated for the new format?

You did a lot of work for #2656 and I know there were some issues but I'd hate to see the baby go with the bath water. Is there something we can do to help recover that?

**Comment by dafriend** [comment]
> Thanks for taking a pass at this! Looks like a test just needs to be updated for the new format?

Here's the fix for the failing unit test.

File: tests/system/Log/ChromeLoggerHandlerTest.php [hunk 1]
```diff
@@ -56,23 +55,26 @@ public function testSetDateFormat()
...
 		$data->code        = 123;
 		$data->explanation = "That's no moon, it's a pumpkin";
 		$result            = $logger->setDateFormat('F j, Y');
 
-		$logger->handle('debug', $data);
-		$peek = $logger->peekaboo();
+		$logger->handle('warning', $data);
 
-		$this->assertEquals($data->explanation, $peek[0]['explanation']);
+		$this->assertTrue($response->hasHeader('x-chromelogger-data'));
```
Comment (lonnieezell) [inline tests/system/Log/ChromeLoggerHandlerTest.php]: Maybe I'm mis-reading something here, but this should actually be false, shouldn't it? If `TestHandler` is only set to log `critical` errors, and we tell the logger to handle a `warning`, then TestHandler (and therefor `ChromeLoggerHandler`) shouldn't send any data, right?

Comment (dafriend) [reply tests/system/Log/ChromeLoggerHandlerTest.php]: The decision to actually log (or not) is made in ``\CodeIgniter\Log\Logger::log()`` and despite the object being called ``$logger`` it is not an instance of ``Logger`` it is an instance of ``ChromeLoggerHandler``.  The severity level of the message has no bearing on this test.

Comment (lonnieezell) [reply tests/system/Log/ChromeLoggerHandlerTest.php]: Doh! You're right. Thanks for clarifying. I'm good with this then.

**Comment by lonnieezell** [comment]
Thanks @dafriend !

**Pull request merged**